src/helix/endpoints/search/mod.rs

```diff
@@ -10,7 +10,7 @@
 //! # let _: &HelixClient<twitch_api::DummyHttpClient> = &client;
 //! # let token = twitch_oauth2::AccessToken::new("validtoken".to_string());
 //! # let token = twitch_oauth2::UserToken::from_existing(&client, token, None, None).await?;
-//! let req = SearchCategoriesRequest::builder().query("Pokémon").build();
+//! let req = SearchCategoriesRequest::query("Pokémon");
```

Comment:
I can't comment there, but this should also point to `HelixClient::search_categories` and say "Manual Request" like the streams.

Maybe all stuff found in `HelixClient` should be referenced in the associated request? Should be a separate PR though.

Reply:
I think we can do this separately